src/mlpack/core/data/dataset_info.hpp

```diff
@@ -101,8 +107,12 @@ class DatasetMapper
     ar & data::CreateNVP(maps, "maps");
   }
 
+  //! Return the policy of the mapper.
   PolicyType& Policy() const;
```

Comment:
I think the return type should be PolicyType const&, else the compiler could not compile(this make sense, const member function return reference non-const data member is weird).